src/lib/openapi/spec/events-schema.test.ts

```diff
@@ -0,0 +1,36 @@
+import { validateSchema } from '../validate';
+import { EventsSchema } from './events-schema';
+
+test('eventsSchema', () => {
+    const data: EventsSchema = {
+        version: 1,
+        events: [
+            {
+                id: 899,
+                type: 'feature-created',
+                createdBy: 'user@company.com',
+                createdAt: '2022-05-31T13:32:20.560Z',
+                data: {
+                    name: 'new-feature',
+                    description: 'Toggle description',
+                    type: 'release',
+                    project: 'my-project',
+                    stale: false,
+                    variants: [],
+                    createdAt: '2022-05-31T13:32:20.547Z',
+                    lastSeenAt: null,
+                    impressionData: true,
+                },
+                preData: null,
+                tags: [],
+                featureName: 'new-feature',
+                project: 'my-project',
+                environment: null,
+            },
+        ],
+    };
```

Comment:
Yeah, that's fair. I just pasted in a semi-random event from the event docs, and this is what it contained. But that's why I want to add more granular event types, because it's really unclear what they are at the moment. I'll see what I can do 🙋🏼